### DISCUSSION TITLE
Get the head repository name in PRs

### DISCUSSION THREAD

**Discussion by ewels** [community/community#26204] [Actions]
<p>The <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables" rel="nofollow noopener">default GitHub Actions environment variables</a> have $GITHUB_HEAD_REF and $GITHUB_BASE_REF for working with pull requests. They also have $GITHUB_REPOSITORY which resolves to the base repository name in pull-requests. However, there does not seem to be any way to get the name of the head repository in PRs (eg. a $GITHUB_HEAD_REPOSITORY).</p>
<p>At the moment we are resolving this by running the following bash command:</p>
<pre><code>$(git remote get-url origin)
</code></pre>
<p>But it seems kind of hacky! Am I missing something here?</p>

**Comment by dawidd6** [comment, marked as answer]
<p>${{github.event.pull_request.head.repo.full_name}}</p>
<p><a href="https://developer.github.com/v3/activity/events/types/#pullrequestevent" rel="nofollow noopener">https://developer.github.com/v3/activity/events/types/#pullrequestevent</a></p>

**Comment by ewels** [comment]
<p>Amazing, thank you! I had not cottoned on to the fact that github.event accessed the whole API like that. So many possibilities!</p>
